src/Mixins/Expectation.php

```diff
@@ -271,16 +272,16 @@ public function toHaveCount(int $count): self
      *
      * @return self<TValue>
      */
-    public function toHaveProperty(string $name, mixed $value = null): self
+    public function toHaveProperty(string $name, mixed $value = new NullValue(), string $failureMessage = ''): self
```

Comment:
@fabio-ivona Can you move this `NullValue` to an folder called `Matchers`, and call it `Any`?

That `Any` type should be created as the following:

`expect($obj)-> toHaveProperty('name', expect()->any(), 'my failure');`

Reply:
done